hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/yaml/YamlProject.groovy

```diff
@@ -4,13 +4,12 @@ package com.linkedin.gradle.azkaban.yaml;
  * Representation of a project in flow.project file for Azkaban Flow 2.0
  */
 class YamlProject {
-  Double azkabanFlowVersion;
+  final Double azkabanFlowVersion = 2.0;
```

Comment:
What about adding the constant 2.0 in AzkabanConstants class?
Something like `static final double AZK_FLOW_VERSION = 2.0`? And assign AZK_FLOW_VERSION to the variable azkabanFlowVersion here?

Reply:
Good idea - I didn't notice the AzkabanConstants class. Thanks!